tests/neg/equality.scala

```diff
@@ -0,0 +1,61 @@
+import dotty.{Eq,EqClass}
+
+object equality {
+
+  case class Str(str: String) extends EqClass[_]
```

Comment:
If you extend `EqClass[_]` it means you have to provide instances of `Eq` yourself; the `eqEq` one won't work.